### ISSUE TITLE
send mail to subscriber with registration details

### ISSUE THREAD

**Issue by samuelstein** [jjbgf/eventbooking#8]

**Comment by numberpi** [comment]
Do you mean sending an e-mail to a newley registered person with his registration information? Or who is the "subscriber"?

**Comment by samuelstein** [comment]
Yes. Sending an e-mail to a newley registered person with his registration information.

Am 9. Februar 2015 22:37:34 MEZ, schrieb Daniel Stolpe notifications@github.com:

> Do you mean sending an e-mail to a newley registered person with his
> registration information? Or who is the "subscriber"?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/jjbgf/eventbooking/issues/8#issuecomment-73595514

## 

Diese Nachricht wurde von unterwegs gesendet.
